app/src/main/java/co/runloop/influencer/adapter/ContactsAdapter.java

```diff
@@ -0,0 +1,88 @@
+package co.runloop.influencer.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import co.runloop.influencer.R;
+import co.runloop.influencer.model.Contact;
+
+public class ContactsAdapter extends RecyclerView.Adapter<ContactsAdapter.ContactViewHolder> {
+
+    private static final String TAG = ContactsAdapter.class.getSimpleName();
+
+    private List<Contact> contacts;
+    private LayoutInflater layoutInflater;
+    private Context context;
+
+    public ContactsAdapter() {
+        contacts = new ArrayList<>();
+    }
+
+    @Override
+    public void onAttachedToRecyclerView(@NonNull RecyclerView recyclerView) {
+        super.onAttachedToRecyclerView(recyclerView);
+        context = recyclerView.getContext();
+        layoutInflater = LayoutInflater.from(context);
+    }
+
+    @NonNull
+    @Override
+    public ContactViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int pos) {
+        View itemView = layoutInflater.inflate(R.layout.item_contact, viewGroup, false);
+        return new ContactViewHolder(itemView);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ContactViewHolder viewHolder, int pos) {
+        viewHolder.bind(contacts.get(pos));
+    }
+
+    @Override
+    public int getItemCount() {
+        return contacts.size();
+    }
+
+    class ContactViewHolder extends RecyclerView.ViewHolder {
+
+        private TextView nameTv;
+        private TextView phoneNumberTv;
+        private ImageView photoImgv;
```

Comment:
It would be nice to keep the terminology consistent. So, either thumbnail or photo, probably better if we don't use both